### PR TITLE
Adds armory protolathe for printing lethal weapons

### PR DESCRIPTION
**Pull request by adamsong**
# General Documentation

### Intent of your Pull Request
Adds an armory protolathe for printing lethal weapons and ammo, removing those items from the security protolathe.

### Why is this change good for the game?
It defeats the purpose of the warden for sec officers to just print guns, bypassing the need to have them issued by the warden.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Lethal weapons and ammo are removed from the sec protolathe, and added to the armory protolathe, located in the back of the armory. The table of what items can be printed in which protolathe will need to be updated to reflect the addition of a new protolathe and the moving of some items to it.

### What general grouping does this PR fall under? 
Security changes

# Changelog

:cl:  
rscadd: Added armory protolathe
/:cl:

### CODE REVIEW

**Comment by Bibby0110** [comment]
Very good.

HoS probably needs both circuits.
Don’t forget the other maps.

**Comment by adamsong** [comment]
Hos gets the armory one, the sec one is in the armory, forgot other maps again

**Comment by alexkar598** [comment]
Placement sucks ass, you can sit in space printing guns without sec being able to do anything except ruin their own printer, destroying the silo link

**Comment by adamsong** [comment]
Where would you suggest it go?

**Comment by alexkar598** [comment]
By moving the implants next to the mindshield implants (where they make more sense to exist anyways), it opens up a spot next to the shotguns (which are used 80% of the time as a hyper lethal gun) where the protolathe can fit.

**Review by patpol4 — CHANGES_REQUESTED** [review]
Omega is now lacking a regular sec lathe, does the armory one include items from it?

**Comment by adamsong** [comment]
Yes, the armory lathe includes sec items, omega armory isn’t big enough to fit the lathe

**Review by patpol4 — APPROVED** [review]
Alright map all good then
